City-of-Helsinki/berth-reservations-ui
# VEN-1404 | Switch application - fetch current berth

## Description :sparkles:
- Pre-populate the form with the user profile data
- Replace the loading spinner with HDS
- Restructure the Redux store to follow the changes
- Fix the logout

## Issues :bug:

### Closes :no_good_woman:

### Related :handshake:

## Testing :alembic:
- Login with the testing user
- Go with the berth application and apply for a `Switch Application`
- On the applicant tap, you will see the current berth information
- Select one of your berths and optionally a reason for switch
- Proceed with the application

### Automated tests :gear:️

### Manual testing :construction_worker_man:

## Screenshots :camera_flash:
![image](https://user-images.githubusercontent.com/23040926/142862702-f606c03b-95a1-437c-af5d-073000d097b7.png)

## Additional notes :spiral_notepad:
